tornjak-frontend/src/tables/table-body.tsx

```diff
@@ -13,16 +14,27 @@ const {
 // rows: rows of data to be rendered on table body
 // getSelectionProps: getSelectionProps func for selecting rows from DataTable
 // returns the body of the table for the specified entity
-class Body extends React.Component {
-    constructor(props) {
+
+type BodyProp = {
+    rows: readonly DenormalizedRow[],
+    entityType: string,
+    getSelectionProps: <E extends object = {}>(data?: ShapeOf<DataTableCustomSelectionData<DataTableRow<string>>, E> | undefined) => ShapeOf<DataTableCustomSelectionProps<DataTableRow<string>>, E> | ShapeOf<DataTableCustomSelectionProps<never>, E>,
```

Comment:
Can we find a better way to define inherited types and props?

Reply:
https://github.com/spiffe/tornjak/issues/58